docs/content/latest/explore/query-1-performance/pg-stat-statements.md

```diff
@@ -138,38 +176,927 @@ View definition:
+rity, c.relhasoids, $1, c.reltablespace, CASE WHEN c.reloftype = $2 THEN $3 ELSE c.reloftype::pg_catalog.regtype::pg_catalo
+g.text END, c.relpersistence, c.relreplident+
+       | FROM pg_catalog.pg_class c                                                                                        
+                                                                                                                           
+                                            +
+       |  LEFT JOIN pg_catalog.pg_class tc ON (c.reltoastrelid = tc.oid)                                                   
+                                                                                                                           
+                                            +
+       | WHERE c.oid = $4
+-[ RECORD 6 ]--------------------------------------------------------------------------------------------------------------
+---------------------------------------------------------------------------------------------------------------------------
+--------------------------------------------
+userid | yugabyte
+dbid   | 12463
+query  | select userid::regrole, dbid, query from pg_stat_statements order by stddev_time desc limit $1
+-[ RECORD 7 ]--------------------------------------------------------------------------------------------------------------
+---------------------------------------------------------------------------------------------------------------------------
+--------------------------------------------
+userid | yugabyte
+dbid   | 12463
+query  | select userid::regrole, dbid, query from pg_stat_statements order by (blk_read_time+blk_write_time)/calls desc lim
+it $1
+-[ RECORD 8 ]--------------------------------------------------------------------------------------------------------------
+---------------------------------------------------------------------------------------------------------------------------
+--------------------------------------------
+userid | yugabyte
+dbid   | 12463
+query  | DISCARD TEMP
+-[ RECORD 9 ]--------------------------------------------------------------------------------------------------------------
+---------------------------------------------------------------------------------------------------------------------------
+--------------------------------------------
+userid | yugabyte
+dbid   | 12463
+query  | select userid::regrole, dbid, query from pg_stat_statements order by total_time desc limit $1
+-[ RECORD 10 ]-------------------------------------------------------------------------------------------------------------
+---------------------------------------------------------------------------------------------------------------------------
+--------------------------------------------
+userid | yugabyte
+dbid   | 12463
+query  | SELECT pg_catalog.pg_get_viewdef($1::pg_catalog.oid, $2)
```

Comment:
I think the output format of the SQL statement is very bad.  Whether the extended display is on or off.  Because this output is related to my local query results.